core/src/reporter/reporter.ts

```diff
@@ -32,6 +32,9 @@ export function reporter(state: State, logger: Logger) {
           if (e.code == OraklErrorCode.DelegatorServerIssue) {
             delegatorOkay = false
           }
+          if (e.code == 'ECONNABORTED') {
```

Comment:
How about hiding then name of error from here and define error in `OraklErrorCode` like `DelegatorTimeoutError`. Additionally, I think it would be better to make it `else if` rather than just `if` to make it more readable. In `core`, we do not use `console`, but Pino logger which is imported to `reporter` function as `logger`.

```typescript
  if (e.code == OraklErrorCode.DelegatorServerIssue) {
    delegatorOkay = false
  }
  else if (e.code == OraklErrorCode.DelegatorTimeoutError) {
    logger.error('Delegator post request timeout')
  }
```